clients/client-java/src/main/java/com/datastrato/gravitino/client/FilesetCatalog.java

```diff
@@ -59,23 +59,27 @@ public com.datastrato.gravitino.file.FilesetCatalog asFilesetCatalog()
   /**
    * List the filesets in a schema namespace from the catalog.
    *
-   * @param namespace A schema namespace.
-   * @return An array of fileset identifiers in the namespace.
+   * @param namespace A schema namespace. This namespace should have 1 level, which is the schema
+   *     name;
+   * @return An array of {@link NameIdentifier} of filesets under the given namespace.
```

Comment:
This PR also has fileset catalog code mixed in. Maybe you need to remove these changes.

Reply:
that is because this PR is based on previous PR, and if previous PR hasn't been merged, its changes also show in this PR; Now the previous PR has been merged, there is no such kind of changes. Please check again.